tests/frameworks/OSXShims/include/IwMalloc.h

```diff
@@ -0,0 +1,67 @@
+//******************************************************************************
+//
+// Copyright (c) 2015 Microsoft Corporation. All rights reserved.
```

Comment:
Your filters are apparently better than mine then D:

---
In reply to: [70346399](https://github.com/Microsoft/WinObjC/pull/634#discussion_r70346399) [](ancestors = 70346399)